project1/application.py

```diff
@@ -39,25 +43,33 @@ def register():
     return render_template('register.html')
 
 
-@app.route("/hello", methods=["POST"])
-def hello():
-    name = request.form.get("Email")
-    pwd = request.form.get("password")
-    print(name+"#"+pwd)
-    return "Hello "+name.split('@')[0]+"! You have successfully registered"
-
-
 # @app.route("/hello", methods=["POST"])
 # def hello():
 #     name = request.form.get("Email")
 #     pwd = request.form.get("password")
-#     ts = time.gmtime()
-#     tstamp = time.strftime("%c", ts)
-#     try:
-#         db.execute("INSERT INTO user_database(username,pword,creation_timestamp)
-# VALUES(:username,:pword,:creation_timestamp)", {
-#             "username": name, "pword": pwd, "creation_timestamp": tstamp})
-#         db.commit()
-#         return "Hello "+name.split('@')[0]+"! You have successfully registered"
-#     except:
-#         return "Failed to Register"
+#     print(name+"#"+pwd)
+#     return "Hello "+name.split('@')[0]+"! You have successfully registered"
+
+
+@app.route("/display", methods=["POST"])
+def display():
+    uname = request.form.get("Email")
+    pwd = request.form.get("password")
+    tstamp = datetime.datetime.now()
+    print(uname, pwd, tstamp)
+    try:
+        db.execute("INSERT INTO userdata(username, passwords, creationstamp) VALUES (:username, :passwords, :creationstamp)",
+                   {"username": uname, "passwords": pwd, "creationstamp": tstamp})
+        db.commit()
+        return "Hello "+uname.split('@')[0]+"! You have successfully registered"
+    except:
+        return "Hello "+uname.split('@')[0] + "! Failed to Register"
+
+
+@app.route("/admin")
+def admin():
+    user_data = Userdata.query.all()
+    lis = []
+    for data in user_data:
+        lis.append(data)
+    return lis
```

Comment:
Following requirement is not met:
The USERS data should be displayed as a bootstrap table with three columns: username, password, creation timestamp.